### PR TITLE
Specify python import paths in manifest

### PR DESCRIPTION
**Pull request by stopping**
* Specify python import paths in manifest

Rather than explicitly modifying PYTHONPATH when running a STL test,
allow manifest to specify 'import_paths' relative to manifest file.

* Add test_driver.RunTest() method

This allows Python to run an STL test directly by providing a manifest
location and manifest arguments.

* Parser returns message fields as dictionary

Return STL message fields as a dictionary for ease of indexing fields
by name.

* Run pyformat

Fixes formatting issues with pyformat.

### CODE REVIEW

File: stl/message.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ class Message(stl.base.NamedObject):
     encoding: An stl.lib.Encoding object which will be used to serialize and
       deserialize MessageValues.
     is_array: Whether this message is an array.
-    fields: List of fields (stl.base.Field) defined in this protocol or message.
+    fields: Dictionary of fields (stl.base.Field) by name defined in this
+       protocol or message.
```
Comment (protobird-git) [inline stl/message.py]: How will you maintain the order of fields? The order will be important for byte stream protocols without field names.

Comment (stopping) [inline stl/message.py]: Decided to revert this for now.

File: test_driver.py [hunk 2]
```diff
@@ -185,18 +191,44 @@ def InitializeStates(transitions):
...
+    self.a_graph = nx.nx_agraph.to_agraph(transition_graph)
+    self.a_graph.layout(prog='dot')
+    for node in self.a_graph.nodes():
+      node.attr['style'] = 'filled'
+      node.attr['fillcolor'] = 'grey'
+    if self.graph_file:
+      self.a_graph.draw(self.graph_file)
+
+  def TransitionRunning(self, edge):
+    self.a_graph.get_edge(edge[0], edge[1], key=edge[2]).attr['color'] = 'green'
```
Comment (protobird-git) [inline test_driver.py]: nice to have comments about edge.

Comment (stopping) [inline test_driver.py]: Done

**Pull request merged**